modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IODispatchers.kt

```diff
@@ -13,6 +15,14 @@ import kotlin.coroutines.CoroutineContext
 object IODispatchers {
   // FIXME use ForkJoinPool.commonPool() in Java 8
   val CommonPool: CoroutineContext = ForkJoinPool().asCoroutineContext()
+
+  private val ioCtr = AtomicInteger(0)
```

Comment:
I would use `AtomicIntW` if available here just in case we decide to use other functions later on